### DISCUSSION TITLE
Does the _build directory need to be in the repository for a jupyterbook?

### DISCUSSION THREAD

**Discussion by kootsoop** [executablebooks/meta#938] [Q&A] [answered]
Working with a collaborator on a book, and we are constantly stepping on each other's toes because of insignificant changes to the _build directory files (primarily .svg files that seem to change IDs every build).

**Comment by rowanc1** [comment, marked as answer]
The build directory is usually ignored from a git perspective, as all files in there should be fully reproducible. See for example the jupyter-book repository code in [`.gitignore`](https://github.com/executablebooks/jupyter-book/blob/master/.gitignore#L74).

There are a few docs in JupyterBook about the [built files](https://jupyterbook.org/en/stable/start/build.html?highlight=_build#aside-source-vs-build-files), if you did read this before and were still confused it would be great to have help improving the docs!

**Reply by kootsoop** [reply]
Will see where I can contribute! Thanks for the response.
